src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -66,12 +66,10 @@ public override async Task RemoveFromGroupAsync(string connectionId, string grou
                 throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(connectionId));
             }
 
-            if (string.IsNullOrEmpty(groupName))
-            {
```

Comment:
Why is this for? We've already have removing user from all groups API. If we want to remove a client connection from all groups, I think it'd be better to have a new API. This class implements interface [HubLifetimeManager<THub>](https://source.dot.net/#Microsoft.AspNetCore.SignalR.Core/HubLifetimeManager.cs) and in original SignalR implementation doesn't allow group name to be null (The `groupName` is not nullable).

Reply:
@xingsy97 failed to update the E2E test so this test will constantly fail before we fixed. I think he is working on fixing the test cases.

I don't know why the previous PR could pass the CI.